app/src/main/java/com/google/droidjump/MainActivity.java

```diff
@@ -20,10 +20,26 @@
 import android.os.Bundle;
 
 public class MainActivity extends AppCompatActivity {
-
+    private int levelsCount;
+    private int currentLevel;
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        levelsCount = 100;
```

Comment:
are these values chosen at random?

Reply:
Yes, just for testing purposes I added hardcoded value. Daria will need to add the right value from a JSON file.